### PR TITLE
add gifs to html report

### PR DESCRIPTION
**Pull request by guidotheelen**
To keep it organised I added the before/after GIFs to the table itself:

image

### CODE REVIEW

File: Lib/diffenator/diff.py [hunk 1]
```diff
@@ -793,15 +799,27 @@ def diff_cbdt_glyphs(font_before, font_after, thresh=4):
...
+                        "diff": diff,
+                        "image": f"<img src='./blaat/{glyph_name_before}.gif'>"
+                    })
+                else:
+                    modified.append({
+                        "glyph before": glyph_name_before,
+                        "glyph after": glyph_name_after,
+                        "string": char,
+                        "diff": diff,
+                    })
```
Comment (RoelN) [inline Lib/diffenator/diff.py:816]: Any way to do this without the repetition? The only difference seems to be the additional `image` key

File: Lib/diffenator/diff.py [hunk 2]
```diff
@@ -779,7 +785,7 @@ def _modified_marks(marks_before, marks_after, thresh=4,
 
 
 @timer
-def diff_cbdt_glyphs(font_before, font_after, thresh=4):
+def diff_cbdt_glyphs(font_before, font_after, thresh=4, rendering=False, html_output=False):
```
Comment (RoelN) [inline Lib/diffenator/diff.py:788]: I don't understand the `rendering` flag, is there a more descriptive name?

Also I see it only used in combination with `html_output` in the if-statements. Can they be combined?

File: Lib/diffenator/diff.py [hunk 3]
```diff
@@ -205,12 +209,14 @@ def mkmks(self, threshold=None):
             thresh=threshold
         )
 
-    def cbdt(self, threshold=None):
-        if not threshold:
+    def cbdt(self, threshold=None, rendering=None, html_output=None):
+        if not threshold or not rendering:
```
Comment (RoelN) [inline Lib/diffenator/diff.py:213]: Originally this sets the treshhold when it's `None`. Now if either one in this if-statement is falsy, they _both_ get reset. Are you sure this is what you want?

File: Lib/diffenator/diff.py [hunk 4]
```diff
@@ -155,10 +157,12 @@ def _to_report(self, limit=50, dst=None, r_type="txt", image_dir=None):
                 elif r_type == "md":
                     reports.append(current_table.to_md(limit=limit))
                 elif r_type == "html":
-                    if image_dir and current_table.renderable:
+                    has_size = self.font_before.size and self.font_after.size
```
Comment (RoelN) [inline Lib/diffenator/diff.py:160]: I just replaced this check with the `self.renderable` flag, but didn't push. Apologies! Please use that one here.

File: Lib/diffenator/diff.py (hunk 1 above)
Comment (guidotheelen) [reply Lib/diffenator/diff.py]: Done ✅

File: Lib/diffenator/diff.py (hunk 2 above)
Comment (guidotheelen) [reply Lib/diffenator/diff.py]: I used the render path now, that clearer. The html output is really something different 🤔

File: Lib/diffenator/diff.py (hunk 3 above)
Comment (guidotheelen) [reply Lib/diffenator/diff.py]: Your right, used a couple statements now ✍️

File: Lib/diffenator/diff.py (hunk 4 above)
Comment (guidotheelen) [reply Lib/diffenator/diff.py]: No problem! 😝

File: Lib/diffenator/diff.py [hunk 5]
```diff
@@ -788,7 +788,7 @@ def _modified_marks(marks_before, marks_after, thresh=4,
 
 
 @timer
-def diff_cbdt_glyphs(font_before, font_after, thresh=4, rendering=False, html_output=False):
+def diff_cbdt_glyphs(font_before, font_after, thresh=4, render_path=None, html_output=False):
```
Comment (RoelN) [inline Lib/diffenator/diff.py:791]: Do we need to pass `html_output` here? Other functions don't need it and always call `modified.report_columns`. Can we do the same here and simplify things a little?

**Pull request merged**